users/views.py

```diff
@@ -1,3 +1,50 @@
-from django.shortcuts import render
+import json, requests, jwt
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+
+from users.models import User
+
+from my_settings import SECRET_KEY, ALGORITHM
+
+class KakaoSignInView(View):
+    def get(self, request):
+        try:
+            kakao_access_token = request.headers.get('Authorization')
+
+            headers = {
+                'Authorization': f'Bearer {kakao_access_token}'
+            }
```

Comment:
```suggestion
            headers = {'Authorization': f'Bearer {kakao_access_token}'}
```
이정도 길이는 한 줄로 쓰는 것이 더 가독성에 좋습니다.

Reply:
넵 수정햇습니다!